gui/pages.py

```diff
@@ -44,7 +42,7 @@ def __init__(self, parent_container, controller, my_bot):
 
 
 class Result(Frame):
-    def __init__(self, parent_container, controller, my_bot):
+    def __init__(self, parent_container, controller, my_bot=None):
```

Comment:
Result is able to not accept my_bot, because in fact my_bot may be useless here... not sure yet